wuvt/templates/auth/login.html

```diff
@@ -8,7 +8,11 @@
 <div class="alert alert-danger">{{ msg }}</div>
 {% endfor %}
 
+{% if config.AUTH_METHOD == 'local' %}
```

Comment:
The login template is only used for local auth, so this conditional is unnecessary. You can actually just remove it and always use ```{{ url_for('auth_local.login') }}```